src/fixate/__main__.py

```diff
@@ -282,16 +282,9 @@ def cancel_tasks():
                 for task in ASYNC_TASKS:
                     task.cancel()
 
-            def finished_test_run_response(future):
-                future.result()
-                self.loop.call_soon(cancel_tasks)
-                self.loop.call_later(1, self.loop.stop)  # Max 1 second to clean up tasks before aborting
-
```

Comment:
I assume this was unused?

Reply:
See other comment.
Effective functionality moved to finished_test_run